pandas/tests/window/common.py

```diff
@@ -348,3 +348,7 @@ def get_result(obj, obj2=None):
         result.index = result.index.droplevel(1)
         expected = get_result(self.frame[1], self.frame[5])
         tm.assert_series_equal(result, expected, check_names=False)
+
+    @staticmethod
```

Comment:
just make this a module level function, ideally we want to get rid of all of the class based stuff eventually here.

Reply:
these also don't need to be private (leading underscore)

Reply:
> just make this a module level function, ideally we want to get rid of all of the class based stuff eventually here.

ahh! I get what you mean! fixed!